tests/mock/test_mock_control_closed_loop.py

```diff
@@ -286,7 +286,7 @@ def test_calc_look_up_forces_temperature(self):
             force_y,
             force_u,
         ) = self.control_closed_loop._calc_look_up_forces_temperature(
-            lut_temperature, 21
+            lut_temperature, 21 * np.ones(12)
```

Comment:
This will change the calculation results in the following assertions, which have the values copied from the M2 cRIO simulator. I would keep it to have the same values at this moment. Thanks!